runtime/lua/vim/uri.lua

```diff
@@ -3,7 +3,7 @@
 -- https://tools.ietf.org/html/rfc2732
 -- https://tools.ietf.org/html/rfc2396
 
-
+local is_windows = vim.loop.os_uname().version:match("Windows")
```

Comment:
Maybe you meant
```suggestion
local is_windows = vim.loop.os_uname().sysname:match("Windows")
```

Reply:
Are there any differences between them?

Reply:
On my end `sysname` is `Linux` to I expect it to be `Windows` on windows machines.

Reply:
Sure. changed.